docs/getting_started/quickstart.ipynb

```diff
@@ -765,7 +772,9 @@
    "source": [
     "## 🤖 Create Deployment\n",
     "\n",
-    "Next step is to deploy the model on KServe behind Hopsworks for real-time inference requests."
+    "Next step is to deploy the model on KServe behind Hopsworks for real-time inference requests.\n",
+    "\n",
+    "**Note:** deployments for scikit-learn without Kserve require a predictor scrip. For details please refer to [Hopsworks documentation](https://docs.hopsworks.ai/latest/user_guides/mlops/serving/predictor/#step-2-optional-implement-predictor-script).  "
```

Comment:
"**Note:** deployments for scikit-learn without Kserve requires a predictor script. For details please refer to [Hopsworks documentation](https://docs.hopsworks.ai/latest/user_guides/mlops/serving/predictor/#step-2-optional-implement-predictor-script).  "